api/commentApi.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import api from "./api";
+
+const loginToken = process.env.NEXT_PUBLIC_LOGIN_TOKEN;
+
+// Comments 조회
+export async function fetchComments(
+  size: number,
+  page: number,
+  mistakeId: string,
+) {
+  try {
+    const response = await api.get("/comment", {
```

Comment:
```suggestion
    const response = await api.get<{comments : CommentDataType[]}>("/comment", {
```
axios 에 타입을 지정해주면 뒤에서 타입 단언을 안해주어도 될 것 같아요